.env

```diff
@@ -0,0 +1,6 @@
+BOOKING_BUG_ENVIRONMENT=treasurydev
```

Comment:
We don't want to make these credentials public. They may be for our test environment, but we don't want the public accessing our test environment.

We've configured VCR to obfuscate these variables but then we've checked them in which makes the VCR configuration pointless.